src/partial.ts

```diff
@@ -0,0 +1,76 @@
+import { purry } from './purry';
+
+type RemovePrefix<
+  T extends ReadonlyArray<unknown>,
+  U extends ReadonlyArray<unknown>,
+> = T extends [...U, ...infer V] ? V : never;
+
+/**
+ * Creates a function that calls `func` with `args` put before the arguments
+ * it receives.
+ *
+ * **Note:** This method doesn't set the `length` property of partially applied
+ * functions.
+ *
```

Comment:
I don't think this note is very important or valuable. It's confusing and not necessarily relevant in a post-ES3/5 world where people rarely care for the "length" of a function.
```suggestion
```